apidef/oas/middleware.go

```diff
@@ -101,12 +104,28 @@ func (c *CORS) ExtractTo(cors *apidef.CORSConfig) {
 }
 
 type Cache struct {
+	// Enabled turns global cache middleware on or off. It is still possible to enable caching on a per-path basis
+	// by explicitly setting the endpoint cache middleware.
+	// Old API Definition: `cache_options.enable_cache`
 	Enabled                    bool     `bson:"enabled" json:"enabled"` // required
+	// Timeout is the TTL for a cached object in seconds.
+	// Old API Definition: `cache_options.cache_timeout`
 	Timeout                    int64    `bson:"timeout,omitempty" json:"timeout,omitempty"`
+	// CacheAllSafeRequests caches responses to (`GET`, `HEAD`, `OPTIONS`) requests overrides per-path cache settings in versions,
```

Comment:
CacheAllSafeRequests caches responses (`GET`, `HEAD`, `OPTIONS`) **to** requests overrides per-path cache

Reply:
Should it be `responses to (GET, HEAD, OPTIONS) requests`?